website/content/accessibility/deaf-and-hard-of-hearing/conference-best-practices.md

```diff
@@ -200,16 +206,13 @@ One of the main reasons people go to conferences is to network. That is no diffe
 
 Networking is key for job opportunities, professional growth, and learning from peers. It is hard enough for people with disabilities to find or change jobs, making these networking events even more important. It is an important channel for professional growth and shouldn't be overlooked in your inclusion efforts. 
 
-Having the opportunity to network with hearing peers is not only important for the individual, it also helps break down prejudices and change attitudes towards us. These one-on-one connections are invaluable for fostering inclusion overall. So please be sure to allow your deaf and hard of hearing attendees and speakers to participate.
-
-When it comes to networking events outside of the official program (e.g., sponsor happy hours), try to find a way to fund interpreters, too. If you can’t use the conference budget, consider creating a sponsorship add-on to fund networking interpreter services. Don’t underestimate the importance of these valuable opportunities.
```

Comment:
This is a bit of wishful thinking. Let's focus on ensuring they provide a great experience during the official program.